include/openssl/core_names.h

```diff
@@ -102,6 +102,23 @@ extern "C" {
 /* Diffie-Hellman Keys */
 #define OSSL_PKEY_PARAM_DH_PUB_KEY   "dh-pub"
 #define OSSL_PKEY_PARAM_DH_PRIV_KEY  "dh-priv"
+/* DSA Parameters */
+#define OSSL_PKEY_PARAM_DSA_P        "dsa-p"
```

Comment:
Do we really need seperate names for dh and dsa?
p, q, g are ffc parameters (for both dsa and dh), maybe we can just name them this way..
One of my old PR's collapsed the params to share the name.

Reply:
So you're suggesting `ffc-p`, `ffc-q`, `ffc-g`?  Sure, I can do that.

Reply:
Done